test/LimitBalance.test.js

```diff
@@ -3,6 +3,12 @@ const { ethGetBalance } = require('./helpers/web3');
 
 const LimitBalanceMock = artifacts.require('LimitBalanceMock');
 
+const BigNumber = web3.BigNumber;
```

Comment:
I don't like doing this pattern if we never use BigNumber by itself again. can we use

```
require('chai')
  .use(require('chai-bignumber')(web3.BigNumber))
  .should();
```

Reply:
Some test files do use it though (but I'd have to check and see if it's necessary). Maybe we could add some sort of global helper everyone requires, that initializes chai and exposes both bignumber and should.

Reply:
actually yeah I was wondering about that; just looked it up and we can do this https://github.com/chaijs/chai/pull/868#issuecomment-261361587 to auto-register should, but we'd have to add a custom one to do bignumber.

Reply:
I think just having that one `require` (and making our setup not so magic) is fine for now.

Reply:
👍

Reply:
I'll add that once this is merged then, to keep the PRs tidy.